test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature

```diff
@@ -0,0 +1,153 @@
+    Then verify that receive a "Created" http code
+    And verify headers in response
+      | parameter      | value                |
+      | location       | /v2/subscriptions/.* |
+      | content-length | 0                    |
+    And verify that the subscription is stored in mongo
+    Examples:
+      | throttling |
+      | 0          |
+      | 1          |
+      | 100        |
+      | 1000       |
+      | 10000      |
+      | 100000     |
+      | 1000000    |
+      | 10000000   |
+
+  @throttling_not_int
+  Scenario Outline:  try to create a new subscription using NGSI v2 with not int type in throttling
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_csub_desc   |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    # These properties below are used in subscriptions request
+    And properties to subscriptions
+      | parameter             | value                   |
+      | subject_idPattern     | ".*"                    |
+      | condition_attrs       | "temperature"           |
+      | notification_http_url | "http://localhost:1234" |
+      | notification_attrs    | "temperature"           |
+      | throttling            | <throttling>            |
+    When create a new subscription in raw mode
+    Then verify that receive a "Bad Request" http code
+    And verify an error response
+      | parameter   | value                    |
+      | error       | BadRequest               |
+      | description | throttling is not an int |
+    Examples:
+      | throttling    |
+      | true          |
+      | ["er", 34]    |
+      | {"value": 34} |
+      | "234324324"   |
+      | 23432.45      |
+
+  @throttling_invalid
+  Scenario Outline:  try to create a new subscription using NGSI v2 with invalid value in throttling
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_csub_desc   |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    # These properties below are used in subscriptions request
+    And properties to subscriptions
+      | parameter             | value                   |
+      | subject_idPattern     | ".*"                    |
+      | condition_attrs       | "temperature"           |
+      | notification_http_url | "http://localhost:1234" |
+      | notification_attrs    | "temperature"           |
+      | throttling            | <throttling>            |
+    When create a new subscription in raw mode
+    Then verify that receive a "Bad Request" http code
+    And verify an error response
+      | parameter   | value                                |
+      | error       | ParseError                           |
+      | description | Errors found in incoming JSON buffer |
+    Examples:
+      | throttling |
+      | 23432,45   |
+      | (23432,45) |
```

Comment:
23432**,**45 != 23432**.**45
the first use a comma (invalid in json) and the second a dot (not allowed in throttling, but valid in json)